practices/dependency-management.md

```diff
@@ -0,0 +1,104 @@
+# Dependency management
+
+The following guide describes how we manage package dependencies in our Node.js-based projects.
+
+
+## Specifying versions of dependencies
+
+As a general rule, we tend to follow a conservative approach when specifying versions for the dependencies of our packages. This helps us ensure consistency between different environments, as we can't know for sure in what environment the app is going to be running, and prevent breakage caused by major or minor updates in dependencies.
+
+We find that the this approach strikes a good balance between the potential breakage caused by less restrictive versions and the maintenance needed when restricting the dependencies to a specific version through the use of `npm shrinkwrap` or similar tools.
+
+We use some [dependency management tools](#dependency-management-tools) to help us keep our apps up to date when new versions of their dependencies are released.
+
+
+### Run-time dependencies
+
+[Run-time dependencies](https://docs.npmjs.com/files/package.json#dependencies) are modules that are required for your application to run, independently of the environment or mode (e.g. development vs production) used.
+
+These are defined in the `dependencies` section of the `package.json` file.
+
+Version numbers for run-time dependencies are defined using a tilde `~` plus a full version number in the MAJOR.MINOR.PATCH form. If the PATCH version is 0, it can be omitted.
```

Comment:
I suggest removing this: "If the PATCH version is 0, it can be omitted."

Should avoid unnecessary exceptions, as it makes for simpler documentation, and in this case will result in less deviation from the norm in the code.